go/oasis-remote-signer/cmd/root.go

```diff
@@ -1,4 +1,4 @@
-// Package cmd implements the commands for the oasis-remote-signer executable.
+// Package cmd implements the commands for the oasis-remotesigner executable.
```

Comment:
This seems incorrect, the previous naming `oasis-remote-signer` was more consistent, change it back.